.github/release.yml

```diff
@@ -3,21 +3,32 @@
 changelog:
```

Comment:
This is for when we generate our release notes. It'll split the commits into sections based on the labels that were on their PRs.

It was already existing from a previous PR so I just fixed it to support the updated labels.